play-services-core/src/main/kotlin/org/microg/gms/auth/signin/AuthSignInService.kt

```diff
@@ -80,7 +80,7 @@ class AuthSignInServiceImpl(
                 val account = account ?: options?.account ?: SignInConfigurationService.getDefaultAccount(context, packageName)
                 if (account != null && options?.isForceCodeForRefreshToken != true && options?.includeUnacceptableScope != true) {
                     if (getOAuthManager(context, packageName, options, account).isPermitted || AuthPrefs.isTrustGooglePermitted(context)) {
-                        val googleSignInAccount = performSignIn(context, packageName, options, account)
+                        val googleSignInAccount = performSignIn(context, packageName, options, account, true)
```

Comment:
silent sign in should not set permitted flag by default (because user did not actively allow sign in)